src/content/en/showcase/2017/santa.md

```diff
@@ -0,0 +1,217 @@
+
+...result in the unsuitability of a naïve offline strategy.
+
+## Santa, built with Polymer
+
+It's worth stepping back and talking about Santa's overall design before diving into how we upgraded it to an offline PWA.
+
+Santa is a single-page application, originally written in Polymer 0.5, and now upgraded to Polymer 1.7.
+Santa is made up of a shared set of code—the router, shared nav assets etc—but it also has many unique 'scenes'.
+
+<figure class="attempt-right">
+  <img src="/web/showcase/2017/images/santa/preloader.gif"  />
+</figure>
+
+Each scene is accessible through a different URL—`/village.html`, `/codelab.html` and `/boatload.html`—and is its own web component.
+When a user opens a scene, we preload all its required HTML and assets (images, audio, css, js), which exist under `/scenes/[[sceneName]]` in the Santa Tracker repository.
+While that happens, users see a friendly preloader that shows progress.
+
+This approach means that we don't have to load unnecessary assets for scenes the user doesn't see (which is a lot of data!).
+It also means that we need to keep an internal 'cache manifest' (keep this in mind—we'll refer to it later) of all the assets required for every scene.
+
+### Load What You Use
+
+This model saves bandwidth, only serving resources required for the scenes a user visits: rather than preloading the whole site at once.
+Santa Tracker leverages Polymer's ability to upgrade custom elements at runtime, rather than at load time-
+
+```html
+<lazy-pages id="lazypages" selected-item="{{selectedScene}}" ... >
+  <dorf-scene id="village" route="village" icon="1f384" permanent
+      mode$="[[mode]]"
+      path$="scenes/dorf/dorf-scene_[[language]].html"
+      class="santa-scene" allow-page-scrolling></dorf-scene>
+
+  <boatload-scene route="boatload" icon="26f5"
+      path$="scenes/boatload/boatload-scene_[[language]].html"
+      loading-bg-color="#8fd7f7"
+      loading-src="scenes/boatload/img/loading.svg"
+      logo="scenes/boatload/img/logo.svg"
+      class="santa-scene"></boatload-scene>
+```
+
+When Santa Tracker first loads, all scenes are effectively treated as unknown elements by the browser.
+The `lazy-pages` element—which inherits behavior from the framework's [`IronSelectableBehavior`](https://www.webcomponents.org/element/PolymerElements/iron-selector/Polymer.IronSelectableBehavior)—listens for a change in the selected page.
+When it changes, it knows to resolve the `path` attribute of a scene, and use HTML imports to load it.
+
+1. The element `<boatload-scene>` is unknown and is treated as a HTMLUnknownElement with some additional attributes
+2. The `<lazy-pages>` element chooses the scene, loading the HTML import `scenes/boatload/boatload-scene_en.html`—this contains the Polymer element and its dependant elements
+3. The friendly preloader is shown
+4. Once the HTML import is loaded and executed, `<boatload-scene>` is transparently upgraded to a real Polymer element, full of holiday cheer 🎄🎉
```

Comment:
I've dropped the previous paragraph and now I've just got the clearly denoted sequence. I think it's clearer. Thanks!